File: collectionview_customlayout_tvguide/customlayout_collectionview/decorationview.m
//
//  decorationview.m
//  customlayout_collectionview
//
//  Created by Rajeeva Ranjan on 02/04/17.
//  Copyright © 2017 Rajeeva Ranjan. All rights reserved.
//

#import "decorationview.h"

@implementation decorationview


- (id)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        // Initialization code
        [self setBackgroundColor:[UIColor lightGrayColor]];
        self.layer.shadowOpacity = 0.5;
        self.layer.shadowOffset = CGSizeMake(0,5);
        self.layer.zPosition = 0;
    }
    return self;
}

@end

File: customlayout_collectionview/decorationview.swift
//
//  decorationview.swift
//  customlayout_collectionview
//
//  Created by Rajeeva Ranjan on 14/10/20.
//  Copyright © 2020 Rajeeva Ranjan. All rights reserved.
//

import Foundation
import UIKit

class decorationview: UICollectionReusableView {
    
    public override init(frame: CGRect) {
        super.init(frame: frame)
        backgroundColor = UIColor.lightGray
        layer.shadowOpacity = 0.5
        layer.shadowOffset = CGSize(width: 0, height: 5)
        layer.zPosition = 0
    }
    
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
}


